prisma/seed.ts

```diff
@@ -54,6 +37,45 @@ function throwIfNotLocalhost() {
   );
 }
 
+async function dropAllData() {
+  // TODO: Programatically clear/truncate all tables, rather than listing each model individually
+  // That way, we won't need to update this method each time we add a new model
+  await Promise.all([
+    prisma.profile.deleteMany({}),
+    prisma.build.deleteMany({}),
+    prisma.salmonRunRecord.deleteMany({}),
+    prisma.freeAgentPost.deleteMany({}),
+    prisma.team.deleteMany({}),
+    prisma.ladderPlayerTrueSkill.deleteMany({}),
+    prisma.ladderMatchPlayer.deleteMany({}),
+    prisma.plusVotingSummary.deleteMany({}),
+    prisma.plusSuggestion.deleteMany({}),
+    prisma.plusStatus.deleteMany({}),
+    prisma.user.deleteMany({})
+  ])
+}
```

Comment:
I figured out how we can get rid of this function. We can add a new script to the package json: `"migrate:reset": "prisma migrate reset",` (check https://www.prisma.io/docs/concepts/components/prisma-migrate)

This resets the DB and runs the seeding script.

Reply:
Oh nice! Yep, that definitely simplifies this code a bit 👍